vue-app/src/views/Landing.vue

```diff
@@ -23,8 +23,9 @@
           <div
             class="apply-callout"
             v-if="
-              $store.state.currentRound &&
-              $store.getters.isRoundJoinPhase &&
+              (($store.state.currentRound &&
+                $store.getters.isRoundContributionPhase) ||
+                $store.getters.isRoundJoinPhase) &&
```

Comment:
We can abstract from the `currentRound` check. It is already being checked by the same getters.

```ts
(isRoundContributionPhase || isRoundJoinPhase) && !isRecipientRegistryFull
```

Note: just trying to simplify as much as we can

Reply:
Checks out!

Reply:
Wait, dont even need the isRoundContributionPhase part in the end. If contributor is in join phase it will cover that, just needed to check that the registry wasnt full.

Reply:
Cool! perfectt